docs/tutorials/simple-maze.md

```diff
@@ -354,7 +354,19 @@ game.onUpdate(function () {
 
 ## Step 10
 
-Change the ``<`` condition in ``||logic:0 < 0||`` to ``<=``. Find the ``||sprites:mySprite x (horizontal position)||`` block and put it in where the first `0` is. Click the dropdown and select ``left``. Put a ``||game:game over||`` inside of ``||logic:if then||``.
+Change the ``<`` condition in ``||logic:0 < 0||`` to ``<=``. Find the ``||sprites:mySprite x||`` block and put it in where the first `0` is. Click the dropdown and select ``left``.
```

Comment:
do we really need to have <= instead of <? Can we drop that step?

Reply:
For simplicity's sake....ok